nuitka/tools/gen_helper_code/BinaryAdd.py

```diff
@@ -0,0 +1,155 @@
+import subprocess
+import os
+from jinja2 import Template
+
+
+def isType(op1, op2):
+	# Operands are compared and checked for their type and value is returned
+	# It checks for operand to be int, float or object
+	if type(op1) == int:
+		if type(op2) == int:
+			return 1
+	elif type(op1) == float:
+		if type(op2) == float:
+			return 0
+	else:
+		return "PyInt_Check(%s)" % op1
+
+
+def add(operand1, operand2):
+	# operands are passed for data type check to isType()
+	operand_type = isType(operand1, operand2)
+	a = str(operand1)
+	b = str(operand2)
+	t = str(operand_type)
+
+	# Subprocess call is used to run C code with passed values 
+	if not os.path.exists('binary_add'):

```

Comment:
This will never exist, but if it did, you wouldn't update your binary, which would be bad.